pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BrokerResourceOnlineOfflineStateModelFactory.java

```diff
@@ -81,6 +82,8 @@ public void onBecomeOnlineFromOffline(Message message, NotificationContext conte
         TableConfig tableConfig = ZKMetadataProvider.getTableConfig(_propertyStore, tableNameWithType);
         _queryQuotaManager.initOrUpdateTableQueryQuota(tableConfig,
             _helixDataAccessor.getProperty(_helixDataAccessor.keyBuilder().externalView(BROKER_RESOURCE_INSTANCE)));
+        _queryQuotaManager.createDatabaseRateLimiter(
```

Comment:
why not use similar method name initOrUpdateDatabaseQueryQuota?

Reply:
Keeping the entry point of create and update flows separate to avoid unnecessary re-computation of broker quota in create flow if the rate limiter already exist.